beancount_import/source/ofx.py

```diff
@@ -967,10 +979,28 @@ def get_subaccount_cash(inv401ksource: Optional[str] = None) -> str:
 
                 cost_spec = None
                 price = None
-                is_sale = False
-                if raw.trantype in SELL_TYPES or (raw.trantype == 'TRANSFER' and
+                is_closing_txn = False
+                if raw.trantype in OPT_TYPES:
+                    assert(raw.optsharesperctrct is not None)
+
+                    if raw.opttradetype.find('TOCLOSE') == -1:
+                        price_per = unitprice * raw.optsharesperctrct
```

Comment:
Here is a bit of the logic behind this code. Each option contract corresponds to some number of shares, usually 100. optsharesperctrct contains that value, so to get the price one actually pays, we need to multiply. Let's look at an example to make sure we are on the same page.
Let's say I trade 1 contract for $0.53. Since the contract corresponds to 100 shares, the actual cost for the transaction is 0.53 * 100 or $53. This is why I name this price_per ... since price is already taken and this is the real money we have transacted for the contract. I'm happy to name it price_per_contract, though in our example this would usually be $0.53. Open to suggesstions since naming is a hard problem :).

Reply:
I would name this price_per_batch.